docker/databricks/build.sh

```diff
@@ -3,6 +3,32 @@
 #
 # Usage: ./build.sh
 
+DATABRICKS_RUNTIME_VERSION="9.1"
+GLOW_VERSION="1.1.2"
+HAIL_VERSION="0.2.85"
+
+# build 9.1 LTS / Spark 3.1.2 images 
+
+# Add commands to build images below
+pushd dbr/dbr$DATABRICKS_RUNTIME_VERSION/
```

Comment:
could add a loop here instead of having the commands twice,

something like this: https://stackoverflow.com/questions/35543088/iterating-over-list-of-arrays-in-bash